src/utils/barcode.ts

```diff
@@ -0,0 +1,60 @@
+import * as bwipjs from "bwip-js";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { tryCatch2v } from "fp-ts/lib/Either";
+import { fromEither, taskEither, taskify } from "fp-ts/lib/TaskEither";
+
+export interface IBarcodeOutput {
+  png: string;
+  pngMimeType: string;
```

Comment:
thre's no need to forward the mime type if the field name is 'png'. the mime type makes sense only if we output a generic list of fields ie.

```ts
attachments: {
  "name": <file name>,
  "data": <content base 64>,
  "mime": <mime type>
}
```

Reply:
ping (remove mime types from here : )